api/v1beta1/flinkcluster_types.go

```diff
@@ -413,6 +413,10 @@ type FlinkClusterSpec struct {
 	// containers.
 	EnvVars []corev1.EnvVar `json:"envVars,omitempty"`
 
+	// Environment variables injected from a source, shared by all JobManager,
+	// TaskManager and job containers.
+	EnvFroms []corev1.EnvFromSource `json:"envFrom,omitempty"`
```

Comment:
The json/yaml field name is not consistent with the Go field name, s/envFrom/envFroms.

Reply:
👍 good catch; updating